.github/workflows/new-milestone-and-issue-in-other-repo.yml

```diff
@@ -0,0 +1,24 @@
+#############################################################################
+# GitHub Action to Create New Milestone and issue in Other Repositories.
+#
+#############################################################################
+name: "R: Update Milestone, Upload '-all' jar, Create issues"
```

Comment:
Let go back to `R: Close/Create Milestone,..`

I do not see upload in action body.

Reply:
Changes Done